docs/can-guides/upgrade/using-codemods.md

```diff
@@ -306,15 +306,15 @@ can.route.start();
 To run [all of the version-4/can-stache transforms](https://github.com/canjs/can-migrate/tree/master/src/transforms/version-4/can-stache) listed below:
 
 ```bash
-can-migrate '**/*.js' -t version-4/can-stache/ -a
+can-migrate '**/*.stache' -t version-4/can-stache/ -a
```

Comment:
I think we should show both because the transforms apply to both types of files.